sdk/confidentialledger/Azure.Security.ConfidentialLedger/src/Generated/ConfidentialLedgerClient.cs

```diff
@@ -453,6 +651,25 @@ public virtual Response GetLedgerEntry(string transactionId, string subLedgerId
         /// <param name="context"> The request context, which can override default behaviors on the request on a per-call basis. </param>
         /// <exception cref="ArgumentNullException"> <paramref name="transactionId"/> is null. </exception>
         /// <exception cref="ArgumentException"> <paramref name="transactionId"/> is an empty string, and was expected to be non-empty. </exception>
+        /// <example>
+        /// This sample shows how to call GetReceiptAsync and parse the result.
```

Comment:
nit/curious: should this say "with required parameters", since we are passing "transactionId" to GetReceiptAsync?  If not, I'd like to understand why since we may need to tune the wording a bit.

Reply:
Done. Add corresponding description when there are only required parameter/request content.